src/DynamicObj/DynamicObj.fsproj

```diff
@@ -32,6 +32,7 @@
   
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Update="FSharp.Core" Version="6.0.1" />
```

Comment:
Why is this needed?

Reply:
There's a bug in the newest SDK because of which it doesn't build. I can remove it as soon as you approve the rest